seleniumJSHttpESModuleEnvTest/src/main/scala/org/scalajs/jsenv/selenium/CamelcaseEsModule.scala

```diff
@@ -0,0 +1,10 @@
+package org.scalajs.jsenv.selenium
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+
+@JSImport("https://cdn.skypack.dev/camelcase@^6.0.0", JSImport.Default)
+@js.native
+object CamelcaseEsModule extends js.Object {
+  def apply(input: String): String = js.native
```

Comment:
Consider a private native member on the `CamelCase` object directly (you might need to upgrade Scala.js for this).

```scala
object CamelCase {
  def hello(...): String = ...

  @JSImport("https://cdn.skypack.dev/camelcase@^6.0.0", JSImport.Default)  
  @js.native
  private def camelCase(input: String): String = js.native
}
```

Reply:
Sure, I think this came in at version 1.1.0. Should I upgrade to that one or go all the way up to 1.2.0?

Reply:
Just create an additional commit that upgrades to the newest Scala.js version. Like this:

https://github.com/scala-js/scala-js-env-selenium/commit/934597103b5ee6185705d928db95f1383de99c34

Reply:
(can be in the same PR)

Reply:
I mean to ask, in that commit, should I upgrade to 1.2.0 or 1.1.0? 
I am not aware of compatibility requirements of this project, hence the question.

Reply:
Just upgrade to the newest version, so 1.2.0. This project does not publish anything that has a dependency on the Scala.js (core). It only has a dependency on `scalajs-jsenv` (which is "below" the core in the dependency trees).

So upgrading Scala.js in this repo does not impose any upgrades on downstream projects.

Reply:
I just realized: We have a bug in our build on these lines: https://github.com/scala-js/scala-js-env-selenium/blob/v1.0.0/build.sbt#L81-L82


`scalajs-js-envs` is *not* release together with Scala.js (anymore). So using the `scalaJSVersion` there does not make any sense. Either hardcode it or introduce `val jsEnvsVersion` (local to the build).

Reply:
So just to be clear, 
- Upgrade Scala.js in [this line](https://github.com/scala-js/scala-js-env-selenium/blob/master/project/build.sbt#L1) to `1.2.0` and
- Upgrade scala-js-env in [these lines](https://github.com/scala-js/scala-js-env-selenium/blob/v1.0.0/build.sbt#L81-L82) to `1.1.1` (as there is no 1.2.0 version of it)

Right?

Reply:
Yes, that is correct.